Code/Framework/AzCore/AzCore/Statistics/StatisticsManager.h

```diff
@@ -63,6 +63,18 @@ namespace AZ
                 }
             }
 
+            void GetAllStatisticsOfUnits(AZStd::vector<NamedRunningStatistic*>& vector, const char* units)
+            {
+                for (auto const& it : m_statistics)
```

Comment:
nitpick: The naming scheme for types are <const-volatile qualifier> <type>
i.e
```suggestion
                for (const auto& it : m_statistics)
```

Reply:
Side-effect of copy/pasting the loop from `GetAllStatistics` just above, will fix both.